ui/src/app.js

```diff
@@ -18,6 +18,9 @@ var isRTL = false
 var two = null
 var cryptomatModel = null
 var termsConditionsTimeout = null
+var termsConditionsAcceptanceInterval = null
+var termsConditionsAcceptanceTimeout = null
+const secondsDelayTermsConditions = 7
```

Comment:
Doesn't this mean that T&C will always take 7 seconds, independently of what is set on `this.trader.termsDelay`? You are sending the `delay` field on the screen transaction in `brain.js`, but that value is never saved. To do so, add an if statement on the `processData()` function, which is where the data sent from `brain.js` functions is processed